src/System.Management.Automation/commands/utility/FormatAndOutput/out-console/OutConsole.cs

```diff
@@ -79,9 +79,14 @@ protected override void BeginProcessing()
                 mrt.MergeUnclaimedPreviousErrorResults = true;
             }
 
-            _savedTranscribeOnly = Host.UI.TranscribeOnly;
+            if (!_transcriptStateChanged)
+            {
+                _savedTranscribeOnly = Host.UI.TranscribeOnly;
+            }
+
             if (Transcript)
             {
+                _transcriptStateChanged = true;
```

Comment:
`Out-Default` lifetime only exists in that pipeline and typically should be the last thing in the pipeline as nothing is emitted if used with `-Transcript`.  Is there a case that's broken?